backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5742980_sys_gh19778_Periods_For_2026.sql

```diff
@@ -0,0 +1,2188 @@
+--- Jan.-26
+
+DO
+$$
+BEGIN
+        IF not exists(select 1
+                      from C_Period
+                      where c_year_id = 540030 and startdate = TO_TIMESTAMP('2026-01-01', 'YYYY-MM-DD') and enddate = TO_TIMESTAMP('2026-01-31', 'YYYY-MM-DD')) THEN
```

Comment:
pls also consider that the year could have another id (manually created before last migration script for example, see https://github.com/metasfresh/metasfresh/commit/d5f3002a68b29e87998a4061d5399d919f55bf6c)

Reply:
thx, considered different year ids